scripts/question.js

```diff
@@ -1312,7 +1312,7 @@ H5P.Question = (function ($, EventDispatcher, JoubelUI) {
         }
       };
 
-      const isSubmitting = extras.contentData
+      const isSubmitting = extras.contentData && extras.contentData.standalone && extras.contentData
```

Comment:
Are the two `extras.contentData` conditions there on purpose?

Reply:
No, removed.